source/testing_govuk_pay/index.html.md.erb

```diff
@@ -5,18 +5,25 @@ weight: 90
 
 # Testing GOV.UK Pay
 
-You will receive a test account for testing in addition to your live credentials.
+When you sign up for GOV.UK Pay, you will receive a test account in addition
+to your live credentials.
```

Comment:
I think the original wording on this was a bit confusing - i think it should be something like 'When we switch your account to live, you will still have a test account' (for testing out settings etc). Basically, you start off with a test account, we create a live account linked with the service, and you still have your original test account. Otherwise, i'm not really sure what's meant here by 'live credentials'....